### ISSUE TITLE
Add support otpauth:// schema url import 

### ISSUE THREAD

**Issue by hydrian** [andOTP/andOTP#167]
Add support to  import OTP consumer from otpauth:// url

https://github.com/google/google-authenticator/wiki/Key-Uri-Format

**Comment by hydrian** [comment]
By android sharing and import by pasting the url after pressing the '+'.

**Comment by hydrian** [comment]
I figured as much. It is just really difficult to import an OTP consumer on an android device when the provider only provides a QR code and you are using that android device. I can use a QR reader to extract the URL from an image file. But their is lots of room for error when trying to manually translate the URL values to andOTP GUI input.

**Comment by X-Coder** [comment]
Maybe importing the image url (right-click QR Code, copy image url, paste in a field) or by saving the qr-image file and importing it would be an option?

Or some kind of screenshot function?

**Comment by hydrian** [comment]
Better yet, let you import an image file of a QR code. You can import an image (jpg, png, gif), andOTP scans the image, extract the URL, and then add it.

**Comment by flocke** [comment]
I like the idea to import image files but I have to look into how to do this with the library I use for QR code processing.

**Comment by JustABanana** [comment]
It would be nice if andOTP automatically set the image from the "issuer" parameter

**Comment by flocke** [comment]
@JustABanana That's a nice idea and should be fairly easily implemented.

**Comment by betsythefc** [comment]
I personally store the backup of all my OTP secrets as optauth:// urls and this would help me by clicking them in my password manager to open and import them into andOTP. Since I found out that andOTP doesn't support this (yet), I moved to saving the secret keys as passwords. It gets the job done but when on a new phone it made it much slower for importing. Though I know I could make a backup and "restore" it to the new phone.
